gui-tk/godley.tcl

```diff
@@ -239,7 +244,11 @@ proc godleyContext {id x y X Y} {
     }
     if {![catch {clipboard get -type UTF8_STRING}]  && ($r!=1 || $c!=0)} {   # Cannot Paste into cell(1,0). For ticket 1064
         .$id.context add command -label "Paste" -command "$id.paste"
-    }
+        
```

Comment:
It works here, because this method constructs and posts the menu immediately after.

However, it would be simpler to create the menu item (via the add command) outside of the conditional, then just have the entryconfigure command inside the conditional (more DRY).

Reply:
This was originally here. If I do  it the other way round, first add command and then if conditional

    if {($r!=1 || $c!=0)} {   # Cannot Paste into cell(1,0). For ticket 1064
        .$id.context add command -label "Paste" -command "$id.paste"
        if {[catch {clipboard get -type UTF8_STRING}]} {
			$id.context entryconfigure end -state disabled 
           }
	}


 I get an error when the clipboard is empty:

invalid command name "minsky.godleyWindow94383534427296.context"
    while executing
"tclunknown minsky.godleyWindow94383534427296.context entryconfigure end -state disabled"

Reply:
That's because of the $id in braces thing again.

Reply:
I don't understand why you're getting this message when the previous line (with add) is working just fine.

Reply:
I fixed this, there was a . missing in front of $id.context entryconfigure end -state disabled.